src/TogglePassword/doc/index.rst

```diff
@@ -0,0 +1,216 @@
+Symfony UX TogglePassword
+=========================
+
+Symfony UX TogglePassword is a Symfony bundle providing visibility toggle for password inputs
+in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+It allows visitors to switch the type of password field to text and vice versa.
```

Comment:
We'll want to add a short gif to the docs right here

Reply:
GIF added 👍🏼